bundle/manifests/ocs-client-operator.clusterserviceversion.yaml

```diff
@@ -482,28 +482,7 @@ spec:
             spec:
               containers:
               - args:
-                - --secure-listen-address=0.0.0.0:8443
-                - --upstream=http://127.0.0.1:8080/
-                - --logtostderr=true
-                - --v=0
-                image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.9.0
-                name: kube-rbac-proxy
-                ports:
-                - containerPort: 8443
-                  name: https
-                  protocol: TCP
-                resources:
-                  limits:
-                    cpu: 500m
-                    memory: 128Mi
-                  requests:
-                    cpu: 5m
-                    memory: 64Mi
-              - args:
-                - --health-probe-bind-address=:8081
-                - --metrics-bind-address=127.0.0.1:8080
                 - --leader-elect
-                - --console-port=9001
```

Comment:
is this change related to kube-proxy and its expected?

Reply:
only the metrics-bind-address, can remove the remaining as well as they are using defaults defined in code.

Reply:
all other options are defaulted and console-port is being set explicitly even though it's the same value